### PR TITLE
Add junit for FindLessonCommandParser

### PR DESCRIPTION
**Pull request by ethantyh692**

### CODE REVIEW

File: src/test/java/tutorease/address/logic/parser/FindLessonCommandParserTest.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+public class FindLessonCommandParserTest {
+
+    private FindLessonCommandParser parser = new FindLessonCommandParser();
+
+    @Test
+    public void parse_emptyArg_throwsParseException() {
+        assertParseFailure(parser, "     ", String.format(MESSAGE_INVALID_COMMAND_FORMAT,
+                FindLessonCommand.MESSAGE_USAGE));
+    }
```
Comment (Ma-Xizi) [inline src/test/java/tutorease/address/logic/parser/FindLessonCommandParserTest.java:22]: Maybe besides empty argument, more types of imvalid arguments could be tested? eg. null, find integers in a name

Comment (ethantyh692) [reply src/test/java/tutorease/address/logic/parser/FindLessonCommandParserTest.java:22]: added for null and find integers

**Review by NorbertLoh — APPROVED** [review]
LGTM!

**Review by Chronoxy — APPROVED** [review]
LGTM 👍

**Pull request merged**